lib/beaker/dsl.rb

```diff
@@ -79,5 +80,10 @@ module DSL
     include Beaker::DSL::Helpers
     include Beaker::DSL::InstallUtils
     include Beaker::DSL::Patterns
+    include Beaker::DSL::PE
+
+    def self.register(helper_module)
```

Comment:
Alright, I've tested this out now and feel pretty good about it after some hemming and hawing; I look forward to expanding this register function!